config/pylama.cfg

```diff
@@ -16,7 +16,7 @@ linters = pylint,pyflakes,radon
 # Note: the following are added to fprime's list of ignored tests for RAPID-0 because of the reasons stated
 ## E0401 is for imports, which we check using our unit testing workflow in order to correctly handle CircuitPython-specific modules
 ## R0903 requires at least two public methods per class, which is not a RAPID-0 style requirement
-disable = C0301,W0612,E0401,R0903
+disable = C0301,W0612,E0401,R0903,R0913
```

Comment:
Can you add a comment indicating what R0913 is so that we remember in the future why it's disabled?